pkgs/test/test/runner/runner_test.dart

```diff
@@ -759,7 +759,7 @@ final foo = true;''').create();
     test('sound null safety is enabled if the entrypoint opts in explicitly',
         () async {
       await d.file('test.dart', '''
-// @dart=2.10
+// @dart=2.11
```

Comment:
These could actually stay at 2.10, it is a bug that it doesn't work

Reply:
Uh...so you approve, but it shouldn't be changed?